### ISSUE TITLE
Image in README

### ISSUE THREAD

**Issue by mattwthompson** [mosdef-hub/foyer#258]
Right now that landing page is a wall of text. There should be a pretty image somewhere in the README - like we have in `mbuild` in order make the landing page visually appealing and provide newcomers a visual memory of `foyer`.

**Comment by ctk3b** [comment]
XML files don't do it for ya? 😁 

How about a diagram of sorts that summarizes the code snippet: 

```
XML + SMARTS ----applied to----> topology = runnable MD input
```

**Comment by mattwthompson** [comment]
Something along those lines sounds good. In particular I was thinking of something that shows how we go from a graph structure and a typeless chemical system (i.e. a single molecule) to a useful system.

**Comment by mattwthompson** [comment]
![image](https://user-images.githubusercontent.com/7935382/61221097-5bd24180-a6dd-11e9-8c67-9e3fb1fb6778.png)


Here's a quick sketch I did in Keynote. Note that I'm just tackling the README image and not a proper Foyer logo.

Let me know what you all think @mosdef-hub/mosdef-maintainers and others

**Comment by ahy3nz** [comment]
1) Is "typed" the common jargon for a chemical system with atomtypes and parameters? Within our group, we use "typed" a bunch, but I somehow get the impression "parametrized" and "non-parametrized" are the more common terms/delineations

2) Can the "typed topology" graphic have some sort of labels of atom types? Otherwise it just looks like we took a 2D skeletal structure and made it 3D without any mention of assigning parameters or atom types

3) I think there should be a mention that the foyer XML is a force field, just in an XML form. So maybe like Foyer Force Field XML? A little verbose, but the label itself doesn't say much about what's the molsim-purpose of the XML unless you look at the contents of the image.

**Comment by ctk3b** [comment]
If you want to get fancy: highlight each atomtype with a specific color and then highlight those patterns on the un-parameterized topology.

**Comment by jennyfothergill** [comment]
resolved by #443
